Player.cpp

```diff
@@ -55,6 +55,52 @@ void Player::showHand() {
 	std::cout << std::endl;
 }
 
+void Player::viewNotebook() {
+	std::string pw;
+	int choice;
+	std::cout << getName() << ", enter your password: ";
+	std::cin >> pw;
```

Comment:
do we have to clear the buffer after this call to cin?

Reply:
i've changed the cin >> pw command to getline(std::cin, pw) so it works with passwords with spaces. i don't think getline() leaves anything behind in the buffer but i'm not 100% sure.